sympy/physics/matrices.py

```diff
@@ -153,6 +154,9 @@ def mgamma(mu, lower=False):
     (0, 0, 0, -1)
 ))
 
+
+@deprecated(issue=20246, useinstead="DFT(n) or DFT(n).as_explicit()",
+            deprecated_since_version="1.7")
```

Comment:
There is a subtle difference that `DFT(n).as_explicit()` is immutable while this returns mutable matrix. So it can be better `Matrix(DFT(n).as_explicit())`

Reply:
Or `DFT(n).as_mutable()`(?). I add that to the deprecation issue if nothing else (since I didn't see your comment before pushing the new version).

Reply:
Yes both can be fine. It’s better to keep the descriptions intact because users may not be familiar the mutability of matrix.